includes/wizards/class-setup-wizard.php

```diff
@@ -277,14 +277,98 @@ public function api_retrieve_theme_and_set_defaults() {
 
 		$theme_mods['header_text']            = get_theme_mod( 'header_text', '' );
 		$theme_mods['header_display_tagline'] = get_theme_mod( 'header_display_tagline', '' );
+
 		return rest_ensure_response(
 			[
-				'theme'      => Starter_Content::get_theme(),
-				'theme_mods' => $theme_mods,
+				'theme'             => Starter_Content::get_theme(),
+				'theme_mods'        => $theme_mods,
+				'homepage_patterns' => $this->get_homepage_patterns(),
 			]
 		);
 	}
 
+	/**
+	 * Load homepage pattern.
+	 *
+	 * @param string $file_path File path of the pattern.
+	 * @return string The pattern markup.
+	 */
+	private function load_homepage_pattern( $file_path ) {
+		$has_donations   = $this->check_service_enabled( 'reader-revenue' );
+		$has_newsletters = $this->check_service_enabled( 'newsletters' );
+		$donate_block    = $has_donations ? '
+		<!-- wp:group {"align":"full","backgroundColor":"primary","className":"newspack-pattern donation__style-3 has-white-color has-text-color"} -->
```

Comment:
This donate block pattern seems to be a duplication of the same `donation__style-3` pattern in Newspack Blocks. Is it possible to check whether Newspack Blocks is active and get the pattern from there if it is, to avoid potential divergence if we revisit the blocks in that plugin?

Reply:
I'd avoid the additional dependency on the patterns - removed these classes in 2d8815ae79cde8b6d69f6d2fa1087a291bb9c2df